apps/checkin/models.py

```diff
@@ -0,0 +1,21 @@
+from django.db import models
+from apps.registration.models import Applicant
+import uuid
+
+
+class CheckInGroup(models.Model):
+    checked_in = models.BooleanField(default=False)
+
+    applicant = models.ForeignKey(Applicant)
+
+    uuid = models.UUIDField(primary_key=True, editable=False, default=uuid.uuid4)
```

Comment:
nit: can you rename this user_id or id, depending on what the field is meant for